www/next.config.js

```diff
@@ -8,6 +8,12 @@ const nextConfig = {
   experimental: {
     largePageDataBytes: 1024 * 1024,
   },
+  rewrites: async () => [
+    {
+      source: "/:any*",
```

Comment:
what's the point of `:any` here?

Reply:
Unsure. Maybe `/:*` is invalid? It's what I copy-pasted from a random blog post on the internet so it must be correct!